src/CycloneDX.Core/Models/Component.cs

```diff
@@ -108,10 +108,20 @@ public enum ComponentScope
         public List<OrganizationalContact> Authors { get; set; }
         public bool ShouldSerializeAuthors() { return Authors?.Count > 0; }
 
-        [XmlElement("author")]
+        [Obsolete("This will be removed in a future version. Use @.authors or @.manufacturer instead.")]
```

Comment:
Nit: Should be manufacture like property name.

Reply:
@thompson-tomo Sorry, I don't understand your comment. The obsolte comment is a one-to-one copy from the specification:
https://cyclonedx.org/docs/1.6/json/#components_items_author

Reply:
interesting that the obsoletion text it has the property "manufacturer" however when u look at the class it is actually "manufacture"

Reply:
In the metadata, there is both "manfacture" (deprecated) and "manufacturer".
In component, you only have "manufacturer".